fem/qspace.cpp

```diff
@@ -16,7 +16,7 @@ namespace mfem
 
 QuadratureSpaceBase::QuadratureSpaceBase(Mesh &mesh_, Geometry::Type geom,
                                          const IntegrationRule &ir)
-   : mesh(mesh_)
+   : mesh(mesh_), order(ir.GetOrder()), size(ir.GetNPoints() * mesh_.GetNE())
```

Comment:
@pazner should size be initialized too?

Reply:
I believe `size` will always be initialized in `ConstructOffsets`